stressberry/cli.py

```diff
@@ -87,13 +110,22 @@ def run(argv=None):
 
     times = []
     temps = []
+    freqs = []
     while t.is_alive():
         times.append(time.time())
-        temps.append(measure_temp(args.temperature_file))
+        temps.append(measure_temp(args.temperature_file, use_vcgencmd))
+        if args.frequency:
+            freqs.append(measure_core_frequency(args.frequency_file, use_vcgencmd))
+            print(
+                "Current temperature: {}°C. Frequency: {}MHz".format(
```

Comment:
It'd be nice to be stricter with the format here to guarantee that all temperatures and frequencies are always right below each other, e.g., `{:3d}` for temp and `{:4d}` for freq.

Reply:
Temperature is a float, when using file it comes to 3 decimal places, whereas vcgencmd it reports `35.0` but I don't think it uses the demical place and always (currently) reports `.0` 

I could convert temp to an int, or report it to 1 decimal place, so that when using file method it can show a tiny bit more resolution.

Reply:
Suggested implementation:
`print("Current temperature: {0:4.1f}°C - Frequency: {:4d}MHz".format(                temps[-1], freqs[-1])`

Reply:
Looking good.